backend/lib/services/members.js

```diff
@@ -52,6 +53,11 @@ function fromResource (project = {}, serviceAccounts = []) {
     .value()
 }
 
+function readProject (client, namespace) {
```

Comment:
duplicate of `projects.readProject`

Reply:
I am not sure if reuse makes sense in this case. Were should this reuse function be located? services, kubernetes-client, cache, entities?

Reply:
will move it to the entities in the future..